css/login-page.css

```diff
@@ -0,0 +1,69 @@
+body{
+    background-color: gray;
+    background-image: url("background-soft4.jpeg");
+    background-repeat: no-repeat;
+    overflow: hidden;
+}
+
+div#forma{
+    width: 300px;
+    margin: 0 auto;
+    padding-top: 40px;
+    display: block;
+    text-align: center;
+}
+
+h1{
+    text-align: center;
+    color: white;
+}
+
+div.jumbotron-fluid{
+    background-color: darkslategray;
+    width: 600px;
+    margin: 80px auto;
+}
+
+img#img-jbt{
+    border-radius: 20px;
+    float: left;
+}
+
+h1#title-j{
+    color: black;
+}
+
+label{
+    text-align: center;
+    color: white;
+}
+
+input{
+    margin: 20px auto; 
+}
+
+div#content{
+    width: 900px;
+    margin: 0 auto;
+    display: block;
+    text-align: center;
+}
+
+button{
+    color: white;
+    background-color: brown;
+    font-size: 25px;
+    padding: 20px 28px;
+    border-radius: 20px;
+
+}
+
+button:hover{
+    color: black;
+    background-color: azure;
+}
+
+button#btn-s{
```

Comment:
Oh remember to avoid abbreviations ex: top-sect => top-section.
The only abbreviations you'll see will be "sm", "md", "xl" ecc... But if you can do not abbreviate